vip-helpers/vip-elasticsearch.php

```diff
@@ -450,4 +450,16 @@ function wpcom_search_api_wp_to_es_args( $args ) {
 	}
 
     return $es_query_args;
-}
\ No newline at end of file
+}
+
+/**
+ * Filter ElasticPress index name if using VIP ES infrastructure
+ */
+function filter_ep_index_name( $index_name, $blog_id, $indexables ) {
+	if ( defined( 'USE_VIP_ELASTICSEARCH' ) && true === USE_VIP_ELASTICSEARCH ) {
+		$index_name = 'vip-' . VIP_GO_APP_ID . '-' . $indexables->slug . '-' . $blog_id;
```

Comment:
`sprintf` would improve the readbility of this line:

```suggestion
		$index_name = sprintf( 'vip-%s-%s-%s', VIP_GO_APP_ID, $indexables->slug, $blog_id );
```